src/landIce/evaluators/LandIce_StokesFOBasalResid_Def.hpp

```diff
@@ -40,11 +40,15 @@ StokesFOBasalResid<EvalT, Traits, BetaScalarT>::StokesFOBasalResid (const Teucho
   w_measure = decltype(w_measure)(p.get<std::string> ("Weighted Measure Name"), dl_basal->qp_scalar);
   normals   = decltype(normals)(p.get<std::string> ("Side Normal Name"), dl_basal->qp_vector_spacedim);
```

Comment:
We may as well move this inside the if statement too.

Reply:
done